modules/sdk-coin-atom/test/unit/transaction.ts

```diff
@@ -38,6 +39,21 @@ describe('Atom Transaction', () => {
       );
       should.equal(Buffer.from(json.signature as any).toString('base64'), testData.TEST_SEND_TX.signature);
       should.equal(tx.type, TransactionType.Send);
+      tx.loadInputsAndOutputs();
+      should.deepEqual(tx.inputs, [
```

Comment:
More of a question since this is the first time i'm seeing this. Are we using `deepEqual` instead of `equal` because we're comparing a custom object with child properties ?

Reply:
thats correct